usecase/src/main/java/com/huchihaitachi/usecase/IsLoggedInUseCase.kt

```diff
@@ -4,5 +4,5 @@ import io.reactivex.Single
 
 interface IsLoggedInUseCase {
 
-    operator fun invoke(): Single<Boolean>
+  operator fun invoke(): Single<Boolean>
```

Comment:
Why `invoke`? 
And why do you need `operator`?

Reply:
Since the use case class has only one purpose (Single Responsibility Principle) it will contain only one method. Using operator fun invoke allows calling a use case by simply adding parenthesis. E.g. val login: LoginUseCase; login(); It makes code more readable.